NitroxClient/Communication/Packets/Processors/InitialPlayerSyncProcessor.cs

```diff
@@ -64,6 +64,7 @@ public override void Process(InitialPlayerSync packet)
 
             SpawnInventoryItemsAfterBasePiecesFinish(packet.InventoryItems, hasBasePiecesToSpawn, packet.PlayerGuid);
             SpawnRemotePlayersAfterBasePiecesFinish(packet.RemotePlayerData, hasBasePiecesToSpawn);
+            SpawnVehiclesAfterBasePiecesFinish(packet.Vehicles, hasBasePiecesToSpawn);
```

Comment:
There's going to be an inherent problem. Cyclops furniture is a basePiece. Spawning cyclops furniture needs to happen after spawning a cyclops.